docs/core/tutorials/libraries.md

```diff
@@ -342,9 +342,9 @@ You may wish to write a library which depends on a native `.dll` file.  If you'r
 For the first option, you'll need to include the following in your `project.json` file:
 
 1. Setting `allowUnsafe` to `true` in a `buildOptions` section.
-2. Specifying the path to the native `.dll`(s) with a [Runtime Identifier (RID)](../rid-catalog.md) under `files` in the `packOptions` section.
+2. Specifying the path to the native `.dll`(s) with a [Runtime Identifier (RID)](../rid-catalog.md) under `runtimes/{RID}/native`, under `mappings`, under `files` in the `packOptions` section.
```

Comment:
For Loc, it'd be better to change `.dll`(s) to `.dll` files or DLLs.
Also, not sure if you'd like to link to these sections you mention on the project.json reference or present this information in a different way? Because the under x, under y, under z format is a bit hard to follow. Maybe show a mock-up of the project.json schema that the person would use?